src/main/resources/query-log-schema-2g.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
This file does not belong here. Please see:

https://github.com/ArpNetworking/metrics-client-doc